angular-app/src/app/workspace/workspace.component.ts

```diff
@@ -104,8 +116,10 @@ export class WorkspaceComponent implements OnInit {
         var playButton = document.getElementById("play");
         var pauseButton = document.getElementById("pause");
 
-        playButton.classList.add("hidden");
-        pauseButton.classList.remove("hidden");
+        if (!this.checkIfPlaying()) {
+            playButton.classList.add("hidden");
+            pauseButton.classList.remove("hidden");
+        }
```

Comment:
Since they're used a few times, you could make adding/removing the "hidden" class to any element into their own hide/reveal functions. Small style/consistency thing, but "hidden" could also be made a constant

Reply:
+1. If using multiple times it is easy to create a `showPlayButton()` and `showPauseButton()` functions that hide one button while showing the other.